content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md

```diff
@@ -233,7 +233,7 @@ Follow the instructions for the cloud provider you are using to archive your log
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```

Comment:
After we finalize the wording here, I'll update all Datadog Agent setup docs (eg. for split logs, log volume control, etc).

Reply:
updated